packages/react-reconciler/src/ReactFiber.old.js

```diff
@@ -772,6 +772,8 @@ export function createFiberFromTracingMarker(
   const tracingMarkerInstance: TracingMarkerInstance = {
     transitions: null,
     pendingBoundaries: null,
+    aborts: null,
+    name: pendingProps.name,
```

Comment:
I thought we said this `name` field wasn't necessary anymore?

Reply:
Talked about it and we decided this was necessary